### PR TITLE
build: inject console and stdout

### PR DESCRIPTION
**Pull request by jazzdan**

### CODE REVIEW

File: internal/build/image_builder.go [hunk 1]
```diff
@@ -45,6 +47,17 @@ func DefaultBuilder(b *dockerImageBuilder) ImageBuilder {
 	return b
 }
 
+func DefaultConsole() console.Console {
```
Comment (nicks) [inline internal/build/image_builder.go]: you should take the io.Writer as a parameter here and let wire take care of it

Comment (jazzdan) [reply internal/build/image_builder.go]: so, `ConsoleFromFile` wants an `*os.File` so I don't think that works

File: internal/build/image_builder.go [hunk 2]
```diff
@@ -402,8 +410,8 @@ func messageIsFromBuildkit(msg jsonmessage.JSONMessage) bool {
 	return msg.ID == "moby.buildkit.trace"
 }
 
-func getDigestFromBuildOutput(ctx context.Context, reader io.Reader) (digest.Digest, error) {
-	aux, err := readDockerOutput(ctx, reader)
+func (l *dockerImageBuilder) getDigestFromBuildOutput(ctx context.Context, reader io.Reader) (digest.Digest, error) {
```
Comment (maiamcc) [inline internal/build/image_builder.go]: we should probs eventually change the name of this receiver, given the struct name change.

Comment (jazzdan) [reply internal/build/image_builder.go]: isn't it still a docker image builder? It's an image builder that builds docker images?

Comment (maiamcc) [reply internal/build/image_builder.go]: ah i mean the letter `l` no longer stands for anything :P

Comment (jazzdan) [reply internal/build/image_builder.go]: ohhh got it, yes will change

**Pull request merged**